backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -53,9 +57,9 @@ public UsersController(WebClientService webClientService, @Value("${config.vanit
         return webClientService.get(usersPath, queryParams);
     }
 
-    @GetMapping("/users/{id}")
-    public Mono<Object> users(@PathVariable String id) {
-        String path = usersPath + "/" + id;
+    @GetMapping("/users/{userId}")
+    public Mono<Object> getUser(@PathVariable String userId) {
```

Comment:
renamed to match the naming of other methods
updated 'id' to be more clear about which id it represents